Readme.md

```diff
@@ -630,4 +630,60 @@ Google's managed service account need following permission in the target (backup
     * Storage Legacy Bucket Reader (```roles/storage.legacyBucketReader```)
         * **NOTE**: it is done automatically set by the ```runner``` service account
     * Storage Legacy Bucket Writer (```roles/storage.legacyBucketWriter```)
-        * **NOTE**: it is done automatically set by the ```runner``` service account
\ No newline at end of file
+        * **NOTE**: it is done automatically set by the ```runner``` service account
+
+## Backups
+
+There are two types of backups supported by Penelope:
+
+* Cloud Storage
+* BigQuery
+  each of them will be described in the following sections.
+
+### Cloud Storage
+
+Source of the backup is a Cloud Storage Bucket. Backups are performed using GCP's Storage Transfer Service. Data are stored in a GCS bucket.
+
+| Feature                 | Oneshot                                 | Snapshot                                 | Mirroring                                |
+|-------------------------|-----------------------------------------|------------------------------------------|------------------------------------------|
+| Backup Frequency        | once                                    | every X hours, min=1                     | every hour                               |
+| Retention Period        | configurable, default=0 (forever)       | configurable, default=0 (forever)        | configurable, default=0 (forever)        |
+| Updated files in source | N/A                                     | Files are replaced                       | Files are replaced                       |
+| Deleted files in source | N/A                                     | Files are kept in backup Bucket          | Files are kept in backup Bucket          |
+| Trashcan support        | no                                      | no                                       | yes                                      |
+| Object versioning       | N/A, default to GCP organization policy | N/A, default to GCP organization policy  | N/A, default to GCP organization policy  |
+
+#### Trashcan for Cloud Storage
```

Comment:
add section for mirror ttl option what it does and general considerations:
Mirror TTL is implemented aa lifecycle rule on sink bucket based on age. In other words, for scenarios where sources from mirror buckets do not undergo regular data changes, a mirror TTL is not recommended.